src/lib/application-state-stores/explorer-stores.ts

```diff
@@ -0,0 +1,81 @@
+import { writable } from "svelte/store";
+import type { MetricsExplorerEntity } from "$lib/redux-store/explore/explore-slice";
```

Comment:
How about we move this type into this file now?

Reply:
Lets remove the redux stuff and do this in there. Didnt touch it in this PR intentionally.